yolov5/baas_utils/error_handling.py

```diff
@@ -54,6 +55,33 @@ def wrapper(*args, **kwargs):
                     # Add the instance to the session
                     session.add(batch_info)
 
+                with db_config.managed_session() as session:
+                    # Subquery to identify rows to be deleted
+                    subquery = session.query(ImageProcessingStatus).join(
+                        DetectionInformation,
+                        and_(
+                            ImageProcessingStatus.image_customer_name == DetectionInformation.image_customer_name,
+                            ImageProcessingStatus.image_upload_date == DetectionInformation.image_upload_date,
+                            ImageProcessingStatus.image_filename == DetectionInformation.image_filename
+                        )
+                    ).filter(
+                        and_(
+                            ImageProcessingStatus.processing_status == "inprogress",
+                            DetectionInformation.run_id == run_id
+                        )
+                    ).subquery()
+
+                    # Perform deletion using the subquery
+                    session.query(ImageProcessingStatus).filter(
+                        and_(
+                            ImageProcessingStatus.image_customer_name == subquery.c.image_customer_name,
+                            ImageProcessingStatus.image_upload_date == subquery.c.image_upload_date,
+                            ImageProcessingStatus.image_filename == subquery.c.image_filename
+                        )
+                    ).delete(synchronize_session=False)
+
+                    LOGGER.info(f"Deleted 'inprogress' rows in ImageProcessingStatus for run_id: {run_id}")
+
```

Comment:
we said that we want to store the errors in the database. Are we doing this now?

Reply:
In this case it could be that the images are corrupted, but where do we store this info about the error?

Reply:
We store this in BatchRunInformation, this is some lines below your comment. It takes the error code of  "except Exception as e:"

Reply:
alright, nice!